lib/api_client/connection/middlewares/request/logger.rb

```diff
@@ -46,7 +46,19 @@ def gather_response_debug_lines(response, taken_sec, debug_lines)
     debug_lines
   end
 
-  def current_stamp_millisec
-    Process.clock_gettime(Process::CLOCK_MONOTONIC, :millisecond)
+  class CurrentTimestamp
+    class << self
+      version = RUBY_VERSION.split('.').map(&:to_i)
```

Comment:
You could use `Gem::Version` class to compare the versions. Sth like:

```ruby
current = Gem::Version.new(RUBY_VERSION)
if current < Gem::Version.new('2.1')
  def millis
...
```

I think it should work :)